### DISCUSSION TITLE
why syntax like rust,golang,dart, kotlin=carbon

### DISCUSSION THREAD

**Discussion by Pgirardin** [carbon-language/carbon-lang#1746] [General]
why syntax like rust,golang,dart, kotlin=carbon

**Comment by josh11b** [comment]
Carbon takes inspiration from a number of languages, attempting to learn from them but without any goal to specifically match any of them. If you would like to talk about specific syntax decisions, I think that would be much more conducive to having a productive discussion than this very broad query.
